wire-ios/Wire-iOS Tests/Mocks/UserSessionMock.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+final class UserSessionMock: UserSession {
```

Comment:
can't we use sourcery to generate this mock? Or is there a good reason to manually maintain this mock?

Reply:
We can but I didn't want to set up sourcery in the sync engine just yet.

Reply:
up to you